configure.ac

```diff
@@ -467,6 +469,11 @@ AC_C_FLAG([-Wbad-function-cast])
 AC_C_FLAG([-Wwrite-strings])
 AC_C_FLAG([-Wundef])
 AC_C_FLAG([-Wimplicit-fallthrough])
+AC_C_FLAG([-Werror=implicit])
+AC_C_FLAG([-Werror=int-conversion])
+AC_C_FLAG([-Werror=incompatible-pointer-types])
```

Comment:
not sure we should be putting any `-Werror` here considering we have the global `-Werror` switch